common/app/views/support/GetClasses.scala

```diff
@@ -47,6 +46,7 @@ object GetClasses {
         ("fc-item--pillar-" + item.pillar.nameOrDefault, true),
         ("fc-item--type-" + item.designType.nameOrDefault, true),
         ("fc-item--pillar-special-report", item.cardStyle == SpecialReport),
+        ("fc-item--pillar-special-report-alt", item.cardStyle == SpecialReportAlt),
```

Comment:
This is just adding the class when `CardStyle` is `SpecialReportAlt`. There is no css targeting this class in this PR. Will be added [in the next one](https://github.com/guardian/frontend/pull/25693).